pkg/client/proxy_snapshot.go

```diff
@@ -92,11 +99,13 @@ func (c *ProxyClient) SnapshotList(serviceAddress string) (snapshotDiskInfo map[
 
 func (c *ProxyClient) SnapshotClone(serviceAddress, name, fromController string) (err error) {
 	if serviceAddress == "" || name == "" || fromController == "" {
-		return errors.Wrapf(ErrParameter, "failed to clone snapshot")
+		return errors.Wrap(ErrParameter, "cannot clone snapshot")
```

Comment:
Should the log be consistent? `failed to clone snapshot` instead?

Reply:
Updated.